apps/api/src/app/environments-v1/usecases/construct-framework-workflow/construct-framework-workflow.usecase.ts

```diff
@@ -246,7 +253,8 @@ export class ConstructFrameworkWorkflow {
       this.logger.error({ err: error }, 'Failed to evaluate skip rule', LOG_CONTEXT);
     }
 
-    return result;
+    // The Step Conditions in the Dashboard control the step execution, that's why we need to invert the result.
+    return workflowOrigin === WorkflowOriginEnum.NOVU_CLOUD ? !result : result;
```

Comment:
Is this necessary? all the workflow executions on construct-framework-workflow.usecase.ts are WorkflowOriginEnum.NOVU_CLOUD, 
in other words, if we are on this "Bridge App" it is WorkflowOriginEnum.NOVU_CLOUD

```suggestion
    return  !result ;
```

Reply:
you are right, I'll change this